TAF/utils/scripts/snap/api-gateway-token.sh

```diff
@@ -1,3 +1,45 @@
-#!/bin/sh
+#!/bin/bash
+ 
 
+option=${1}
+# Command vars
+USER="gateway"
+GROUP="gateway-group"
 
+# JWT File
+JWT_FILE=/var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt
+JWT_VOLUME=/var/snap/edgexfoundry/current/secrets/security-proxy-setup
+
+mkdir -p ${WORK_DIR}/keys
+
+if [ ! -f "${WORK_DIR}/keys/${USER}.pub" ];
```

Comment:
This looks like it's allowing you to use pre-configured keys. Is there any TAF documentation on how to set this up? If so, a comment would be helpful as well as a pointer to the doc(s) for this (if they exist).

Reply:
No, the keys are just generated there in the TAF working directory and stored between test runs, so they would not normally be pre-configured.